src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/IAnalyzeResultProvider.cs

```diff
@@ -2,14 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.DotNet.UpgradeAssistant
 {
     public interface IAnalyzeResultProvider
     {
+        string AnalysisTypeName { get; }
+
         [ObsoleteAttribute("This property is WIP, expect changes in this area.", false)]
```

Comment:
Could this attribute be moved up to the interface if you want to keep it?

Reply:
I'll remove it with this change as now customers can get a version of end-to-end analysis experience.